src/options.h

```diff
@@ -29,16 +30,15 @@ class options_manager
                     : std::pair<std::string, translation>( first, second ) {
                 }
         };
-        static std::vector<id_and_option> lang_options;
-        static std::vector<id_and_option> actual_lang_options;
+        static const std::vector<id_and_option> actual_lang_options;
```

Comment:
Having got rid of `lang_options`, you might as well rename this new variable back to just `lang_options` again.